src/ir_Mitsubishi.h

```diff
@@ -88,7 +89,9 @@ union Mitsubishi144Protocol{
     // Byte 14
     uint8_t          :8;
     // Byte 15
-    uint8_t          :8;
+    uint8_t          :5;
+    uint8_t iSave10C :1;  //i-SAVE: 1 if in Heat mode AND iSave os on, and temperature THEN set to 10C
```

Comment:
Needs a space after comment chars.
e.g.
`    uint8_t iSave10C :1;  // i-SAVE: 1 if in Heat mode AND iSave os on, and temperature THEN set to 10C`